app/src/contexts/FolderContext.ts

```diff
@@ -0,0 +1,16 @@
+import { createContext, useContext } from "react"
+
+export interface Path {
+  path: string
+  setPath: (path: string) => void
```

Comment:
Can we type `setPath` as `Dispatch<SetStateAction<string>>` instead? That is the type of a `setState` update function. This can be seen by hovering over `setPath` in `Providers.tsx`